src/schrodinger/sketcher/molviewer/scene.cpp

```diff
@@ -261,33 +276,14 @@ void Scene::onDisplaySettingsChanged()
             RDKit::common_properties::molNote, note);
         simplified_stereo_annotation = QString::fromStdString(note);
     }
+
     m_simplified_stereo_label->setPlainText(simplified_stereo_annotation);
     m_simplified_stereo_label->setVisible(
         !simplified_stereo_annotation.isEmpty());
     m_simplified_stereo_label->setPos(
         getInteractiveItemsBoundingRect().bottomLeft());
-
     m_fonts.setSize(m_sketcher_model->getFontSize());
 
-    // we need to update all of the atom items before we update any bond
-    // items, since bond items pull information from their associated atom
-    // items
-    for (auto item : getInteractiveItems(InteractiveItemFlag::ATOM)) {
-        // if the scene displays a simplified stereo annotation, we hide the
-        // atomic ones
-        auto atom_item = static_cast<AtomItem*>(item);
-        atom_item->setHideStereoLabels(!simplified_stereo_annotation.isEmpty());
-        atom_item->updateCachedData();
-    }
-    for (auto item : getInteractiveItems(InteractiveItemFlag::MONOMER)) {
-        static_cast<AbstractAtomOrMonomerItem*>(item)->updateCachedData();
-    }
-
-    for (auto item :
-         getInteractiveItems(InteractiveItemFlag::BOND_OR_CONNECTOR)) {
-        static_cast<AbstractBondOrConnectorItem*>(item)->updateCachedData();
-    }
-
```

Comment:
I think we need this code to get the image to update after changing settings.  E.g., if you change the font size, do the atoms update correctly without these `updateCachedData` calls?